bugbug/bug_features.py

```diff
@@ -481,6 +484,13 @@ def __call__(self, bugs, **kwargs):
         return bugs[0]["target_milestone"] == bugs[1]["target_milestone"]
 
 
+class is_first_affected_same(couple_bug_feature):
+    def __call__(self, bugs, **kwargs):
+        return min(
+            map(int, affected_then_unaffected().get_lists(bugs[0], **kwargs)[1])
```

Comment:
Not sure you need to convert to int, but if you do do it directly in the helper function when you get the list of affected versions.

Reply:
Oh, I just realized comparison operations work between strings, so I guess there's no need for the `map(int ..)`